translations/translations.json

```
@@ -4763,7 +4763,12 @@
       "positiveRating": "Diese Seite ist hilfreich",
       "searchTermDescription": "Für den folgenden Begriff konnte kein Ergebnis gefunden werden:",
       "contactMailAddress": "E-Mail für Rückfragen",
-      "note": "Bitte wähle eine Reaktion oder schreibe einen Kommentar, um das Feedback abschicken zu können."
+      "note": "Bitte wähle eine Reaktion oder schreibe einen Kommentar, um das Feedback abschicken zu können.",
+      "giveFeedback": "Feedback geben",
+      "checkQuery": "Überprüfen Sie Ihren Suchbegriff oder ändern Sie die eingestellte Sprache der {{appName}}-App.",
+      "informationMissing": "Hier fehlen Informationen?",
+      "noResultsInOneLanguage": "Es wurden leider keine passenden Ergebnisse in Deiner Sprache gefunden.",
+      "noResultsInTwoLanguages": "Es wurden leider keine passenden Ergebnisse in Deiner Sprache oder auf Deutsch gefunden."
```

:wrench: I think `oneLanguage` and `twoLanguage` are not really telling. Instead I would just do something like
```suggestion
      "noResultsInUserLanguage": "Es wurden leider keine passenden Ergebnisse in Deiner Sprache gefunden.",
      "noResultsInUserAndSourceLanguage": "Es wurden leider keine passenden Ergebnisse in Deiner Sprache oder auf Deutsch gefunden."
```

---

:thinking: If we ever change the source language, for example for a new whitelabel in some other country, this kinda breaks. Since this is not planned at the moment and we have translation overrides, this is probably okay. But definitely something to keep in mind for the future. Perhaps `Originalsprache` would also work? But probably not as clear/easily understandable.